### ISSUE TITLE
Bioperl update causing tests to fail

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse#1310]
Need to investigate if this causes actual code in normal use cases to fail via setup.sh

Recent releases here https://metacpan.org/release/BioPerl

**Comment by cmdcolin** [comment]
Fails https://travis-ci.org/GMOD/jbrowse/jobs/489606291

Same commit several weeks ago succeeded https://travis-ci.org/GMOD/jbrowse/builds/487255044

**Comment by mortimersebastian** [comment]
I am having this same issue on macOS Sierra.

**Comment by cmdcolin** [comment]
@mortimersebastian thanks for reporting. This bioperl update affected quite a few things, there are some basic functions that you can still use in JBrowse though e.g. prepare-refseqs.pl, flatfile-to-json.pl and generate-names.pl should still work

Note that if you want full functionality, you can manually install the older version of BioPerl

Example

```
rm -rf extlib # inside the jbrowse directory
cpanm https://github.com/bioperl/bioperl-live/archive/release-1-7-2.tar.gz
./setup.sh
```

I think this will get the older bioperl and keep it through the installation

**Comment by keiranmraine** [comment]
FYI, I had to add the following to build the new BioPerl:

```
   - libxml2-dev
   - libgd-dev
   - libdb-dev
```

Looks like they've merged some bits back in,

**Comment by cmdcolin** [comment]
@keiranmraine which version of bioperl are you referring to? BioPerl 1.7.3+ underwent a big change where many core BioPerl packages were moved into separate ones. see changelog here https://metacpan.org/release/CDRAUG/BioPerl-1.7.3

One module, Bio::DB::SeqFeature::Store which jbrowse depends on had this happen so setup.sh does not get it currently, and actually Bio::DB::SeqFeature::Store still needs uploading to cpan.

**Comment by keiranmraine** [comment]
It may have been the change coincided with changes to the underlying travis build hosts, we had lots of things suddenly start failing.

**Comment by cmdcolin** [comment]
Attempted fix is on dev, pins to Bio::Perl@1.7.2. The method can be fooled but should be better than what's currently there which is just sorta broken

Proper fix would be to upload some modules that were lost in translation but unknown timeline on that
